.github/ISSUE_TEMPLATE/new-language.md

```diff
@@ -0,0 +1,22 @@
+---
+name: new language
+about: Request a new language for onefetch to support
+title: "[NEW LANGUAGE]"
+labels: 'Type: Enhancement, good first issue'
```

Comment:
```suggestion
labels: 'Type: enhancement, good first issue'
```

Reply:
I think `Enhancement` has to be capitalized, since the issue in this repo is `Type: Enhancement`.